src/Sylius/Bundle/ApiBundle/DataProvider/PaymentRequestDataProvider.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\DataProvider;
+
+use ApiPlatform\Core\DataProvider\ItemDataProviderInterface;
+use ApiPlatform\Core\DataProvider\RestrictedDataProviderInterface;
+use Sylius\Bundle\ApiBundle\Checker\UpdatePaymentRequestEligibilityCheckerInterface;
+use Sylius\Bundle\ApiBundle\Serializer\ContextKeys;
+use Sylius\Component\Payment\Model\PaymentRequestInterface;
+use Sylius\Component\Payment\Repository\PaymentRequestRepositoryInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+/** @experimental */
+final class PaymentRequestDataProvider implements RestrictedDataProviderInterface, ItemDataProviderInterface
```

Comment:
```suggestion
final class PaymentRequestUpdateDataProvider implements RestrictedDataProviderInterface, ItemDataProviderInterface
```
Since it's the only case within this service